File: OpenGL-ES2.0/OpenGL-ES2.0/ViewController.m
//
//  ViewController.m
//  OpenGL-ES2.0
//
//  Created by TuLigen on 16/9/21.
//  Copyright © 2016年 TuLigen. All rights reserved.
//

#import "ViewController.h"
#import "ES2.0/ESRenderView.h"

@interface ViewController ()
{
    ESRenderView *_renderView;
}
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    _renderView = [[ESRenderView alloc] initWithFrame:self.view.bounds];
    
    [self.view addSubview:_renderView];
}
-(void) viewWillDisappear:(BOOL)animated
{
    [_renderView viewWillDisappear];
}
- (IBAction)userCamera:(id)sender {
}
- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
- (IBAction)popListUI:(id)sender {
    
}
- (IBAction)popView:(id)sender {
}

@end

File: SwiftDemo/SwiftDemo/ViewController.swift
//
//  ViewController.swift
//  SwiftDemo
//
//  Created by TuLigen on 16/2/17.
//  Copyright (c) 2016年 TuLigen. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var helloBtn: UIButton!
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    @IBAction func clicked(sender: AnyObject) {
        var alert = UIAlertController(title: "hello", message: "hello world", preferredStyle: UIAlertControllerStyle.Alert);
        alert.addAction(UIAlertAction(title: "close", style: UIAlertActionStyle.Default, handler: nil))
        self.presentViewController(alert, animated: true, completion: nil)
        self.helloBtn.setTitle("CC", forState: UIControlState.Normal);
    }
}

